build.gradle

```diff
@@ -11,11 +11,55 @@ repositories {
 }
 
```

Comment:
also add a publish configuration so that this plugin can be pulished to the gradle plugin repository

Reply:
Would have added the publish configuration when setting up the pipelines in the next PR